src/components/Auth/NewVerificationForm.tsx

```diff
@@ -0,0 +1,44 @@
+'use client'
+
+import { newVerification } from '@/lib/auth/new-verification'
+import { useSearchParams } from 'next/navigation'
+import { useCallback, useEffect, useState } from 'react'
+import Alert from '../Alert/Alert'
+import { Loading } from '../Loading/Loading'
+
+export const NewVerificationForm = () => {
+  const [error, setError] = useState<string | undefined>()
```

Comment:
error i success razem z tymi alertami czesto sie powtarza, takze moze daloby sie to wydzielic?